Dockerfile

```diff
@@ -3,7 +3,8 @@ MAINTAINER Brian Prodoehl <bprodoehl@connectify.me>
 
 ENV HOME /root
 CMD ["/sbin/my_init"]
-EXPOSE 80 443
+EXPOSE 80
```

Comment:
This syntax was right before, and I don't see a need for an extra layer in the image just to put these on their own line. http://docs.docker.com/reference/builder/#expose